### DISCUSSION TITLE
Integration with Hub

### DISCUSSION THREAD

**Discussion by smastelini** [online-ml/river#527] [Ideas]
[Hub](https://github.com/activeloopai/Hub) is a nice tool for big data management. @mikayelh and the other Hub team members have demonstrated an interest in providing tools to make river get data from Hub.

They currently generate numpy arrays, but using pandas should also be viable. Hence, Hub could also be a source of data for our mini-batching capable models (or the ones that learn instance by instance, who knows).

@mikayelh, feel free to expose any ideas that you have or introduce other members of your team!

**Comment by mikayelh** [comment]
@smastelini thank you so much for kickstarting this - we're excited to explore cool stuff we can do together! I'm including @davidbuniat @mynameisvinn and @imshashank from our team so they can share their thoughts!

**Reply by mynameisvinn** [reply]
@smastelini Nice meeting you! Perhaps we can do a screen share next week so I can show you how Hub streams data?

**Reply by smastelini** [reply]
Yeah! Let's do that! We can continue this planning via e-mail :)
